Exercise-24/index.js

```diff
@@ -1,13 +1,13 @@
 "use strict"
 //for loop - adding up nubmers 0-1000 break when result is more than 400
 
-let var1 = 0;
-let var2 = 0;
-for (var2 = 0; var2 <= 1000; var2++) {
-    var1 += var2;
-    console.log(var1);
+let firstAddend = 0;
+let secondAddend = 0;
```

Comment:
This line is unnecessary 🙂 
going forward, when writing code, after creating a pull request, make sure, you don't leave any code that is redundant